packages/accordions/src/elements/stepper/components/Step.tsx

```diff
@@ -9,6 +9,9 @@ import React, { forwardRef, useRef, HTMLAttributes } from 'react';
 import { StyledStep, StyledLine } from '../../../styled';
 import { StepContext, useStepperContext } from '../../../utils';
 
+/**
+ * @extends HTMLLIElement
```

Comment:
Should be `HTMLAttributes<HTMLLIElement>`

Reply:
Fixed.

Reply:
extends have been removed for now.